test/unittest/meson.build

```diff
@@ -16,6 +16,7 @@ unzip_target = [
   ['unittest_layers_v2.tar.gz', 'unittest_layers'],
   ['unittest_models.tar.gz', 'unittest_models'],
   ['unittest_models_v2.tar.gz', 'unittest_models'],
+  ['unittest_models_v3.tar.gz', 'unittest_models'],
```

Comment:
How about making a another file which will include unittest grucellCellFC so that we can make 1 tar.gz files from 1 python script. I think this will make it more manageable.

Reply:
Actually, this golden data needs to be included in unittest_models_v2.tar.gz. But there is a file size limitation, there is no choice but to create v3.tar.gz. If we make another golden data later on, then we have to insert it to `unittest_models_v3.tar.gz`

Reply:
That is a good idea. I will make an issue though.